aburrell/apexpy
# Memory error

Description
===========

This is a draft PR to make it easier to track some of the issues/changes pertinent to #115.  The main problem was a variety of memory errors that seemed to occur somewhat randomly when an `Apex` object was initialized, but there seem to be several things that potentially could contribute to this. (See #115 for ongoing discussion - will copy main points to this thread when ready to review).

Fixes #115 (not yet)

Type of change
--------------
Please delete options that are not relevant.

* Bug fix (non-breaking change which fixes an issue)

How Has This Been Tested?
-------------------------
* Confirm that short example scripts can be run.
* Check that the fortran test program compiles and generates the correct output.
* Standard unit tests

### Test Configuration

* Operating system: MacOS 12.6.1 M1
* Python version number: 3.10.9
* Compiler with version number: gfortran 12.2.0

Checklist
---------
- [ ] Add a note to ``Changelog.rst``, summarising the changes
- [ ] Add yourself to ``AUTHORS.rst`` and ``.zenodo.json``
